recipes/xia2/meta.yaml

```diff
@@ -0,0 +1,116 @@
+    - dev.xia2.check_mosaic = xia2.cli.check_mosaic:run
+    - dev.xia2.create_mask = xia2.cli.create_mask:run
+    - dev.xia2.file_statistics = xia2.cli.file_statistics:run
+    - dev.xia2.make_sphinx_html = xia2.cli.make_sphinx_html:run
+    - dev.xia2.show_mask = xia2.cli.show_mask:run
+    - dev.xia2.show_mtz_cells = xia2.cli.show_mtz_cells:run
+    - xia2.ssx = xia2.cli.ssx:run
+    - xia2.ssx_reduce = xia2.cli.ssx_reduce:run
+    - xia2.add_free_set = xia2.cli.add_free_set:run
+    - xia2.compare_merging_stats = xia2.cli.compare_merging_stats:run
+    - xia2.delta_cc_half = xia2.cli.delta_cc_half:run
+    - xia2.get_image_number = xia2.cli.get_image_number:run
+    - xia2.html = xia2.cli.xia2_html:run_with_log
+    - xia2.index = xia2.cli.index:run_with_log
+    - xia2.integrate = xia2.cli.integrate:run_with_log
+    - xia2.is_doing = xia2.cli.is_doing:main
+    - xia2.ispyb_json = xia2.cli.ispyb_json:run
+    - xia2.ispyb_xml = xia2.cli.ispyb_xml:run
+    - xia2.merging_statistics = xia2.cli.merging_statistics:run
+    - xia2.multiplex = xia2.cli.multiplex:run
+    - xia2.npp = xia2.cli.npp:run
+    - xia2.overload = xia2.cli.overload:run
+    - xia2.plot_multiplicity = xia2.cli.plot_multiplicity:run
+    - xia2.print = xia2.cli.print:run_with_log
+    - xia2.rebatch = xia2.cli.rebatch:run_with_log
+    - xia2.report = xia2.cli.report:run_with_log
+    - xia2.rescale = xia2.cli.rescale:run_with_log
+    - xia2.setup = xia2.cli.setup:run
+    - xia2.small_molecule = xia2.cli.small_molecule:run
+    - xia2.strategy = xia2.cli.strategy:run_with_log
+    - xia2.table1 = xia2.cli.table1:run
+    - xia2.to_shelx = xia2.cli.to_shelx:run
+    - xia2.to_shelxcde = xia2.cli.to_shelxcde:run
+    - xia2 = xia2.cli.xia2_main:run
+    - xia2.cluster_analysis = xia2.cli.cluster_analysis:run  
+
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - cctbx-base >=2023.10
+    - dxtbx ={{ version_xy }}
+    - dials ={{ version_xy }}
+  run:
+    - {{ pin_compatible('cctbx-base') }}
+    - {{ pin_compatible('dxtbx', max_pin='x.x', lower_bound=version_xy) }}
+    - {{ pin_compatible('dials', max_pin='x.x', lower_bound=version_xy) }}
+    - python >=3.9
+    - dials-data >=2.0
+    - jinja2
+    - pyyaml
+    - tabulate
+
+test:
+  source_files:
+    - xia2/tests
+    - xia2/conftest.py
+  imports:
+    - xia2
+  requires:
+    - dials-data
+    - pytest
+    - pytest-mock
+    - pip
+  commands:
+    - >
+      pytest xia2/tests/ --regression-full
+      --ignore=xia2/tests/command_line/test_to_shelx.py
+      --ignore=xia2/tests/Wrappers/Dials/test_dials_wrappers.py
+      --ignore=xia2/tests/Applications/test_xia2setup.py
+      --ignore=xia2/tests/regression/test_cluster_analysis.py
+      --ignore=xia2/tests/regression/test_overload.py
+      --ignore=xia2/tests/regression/test_multiplex.py::test_prot_k_multiwave_double
+      --ignore=xia2/tests/regression/test_ssx.py  # [win]
+    - pytest xia2/tests/ --regression-full  # [not win]
```

Comment:
On not-win this will run both commands, minus the last argument of the first command, and therefore run tests twice. Is that on purpose?

Reply:
Good catch. Probably better to check the target platform and run tests accordingly...

Reply:
If you know the test doesn't work on Windows can't you just mark it up
```python
@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
```
It'd be better to have this outside of the conda-forge recipe to be honest, if the reason the test isn't working on a specific platform has nothing to do with conda-forge.

Reply:
Thanks for the suggestion, Markus. Will add a patch!